engine/orchestration/src/main/java/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -333,6 +335,8 @@
     private NetworkOfferingDetailsDao networkOfferingDetailsDao;
     @Inject
     private NetworkDetailsDao networkDetailsDao;
+    @Inject
+    private SecurityGroupManager _securityGroupManager;
```

Comment:
[Java Naming Conventions](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html) does not recommend using _ (underscore) before variable name. Can you please remove it?